cli/cmd/cmd.go

```diff
@@ -164,6 +164,9 @@ func New(driver api.API, config utils.ConfigReader, logControl logging.LogContro
 		cli.StringFlag{"auth0-scope", defaultOps.Auth0Scope, "Scope to request in Auth0"},
 		cli.StringFlag{"keyproxy-json-server", defaultOps.KeyProxyJsonServer, "URL for API key server (cc auth token endpoint)"},
 		cli.StringFlag{"keyproxy-listen-port", defaultOps.KeyProxyListenPort, "Port for API key proxy to listen on"},
+		cli.IntFlag{Name: "instances", Usage: "Service instance count"},
```

Comment:
AFAIK, the flags in this section are global to all commands. We don't want these flags to be global to all.  I think what you added to `cli/cmd/service.go` should be sufficient

Reply:
Works for me.